Automattic/crowdsignal-ui
# Bump isolated-block-editor to version 2.3.0

This patch bumps the version of `isolated-block-editor` to `2.3.0` which adds a `<ToolbarSlot />` component that allows for inserting items into the editor toolbar.

As part of this PR, I also manually bumped all our dependencies on `@wordpress/...` libraries to match the ones used in `isolated-block-editor`.  
This solves the issue with multiple versions of the same library being installed causing issues for our custom blocks (see #26). Eventually we'll want to make them external altogether but for now this will do.

Another change is how now the styles need to be imported manually.

# Testing

- Run `yarn install`
- Run `yarn workspace @crowdsignal/dashboard start` and go to `https://crowdsignal.localhost:9000/edit/poll/123`
- The editor should still load normally with the correct styles.
